.gitlab-ci.yml

```diff
@@ -66,9 +66,19 @@ variables:
   SPECIFIC_INTEGRATION_TEST: ""
   TESTS_IN_PARALLEL: "4"
 
+  # Child pipelines build and test
+  # TODO: Should "TEST" assume "BUILD"?
```

Comment:
IMO the separation would be helpful if we run this pipeline with a build artifact from another pipeline - not sure if this is considered anytime soon for these components though...

Reply:
I like it too, and unless someone opposes I leave it.

The detail here is that in the implementation the build will be triggered when `PUBLISH_RELEASE_AUTOMATIC` is set, even if these `BUILD_*` are not set. So to be more honest with what is happening it could be removed and just use "TEST or PUBLISH" as trigger for BUILD. (does it make sense?)

Reply:
Indirectly commented below by Kristian. I'll clean it up but the explicit BUILD_ variables stay :+1: